scripts/gen_eval_files.lua

```diff
@@ -211,6 +211,39 @@ local function get_api_meta()
   return ret
 end
 
+--- @param text string
+local function fix_indent(text)
```

Comment:
A description of what this function does would be useful. How is the indent being _fixed_?